manager/assets/modext/workspace/package.grid.js

```diff
@@ -148,8 +148,19 @@ MODx.grid.Package = function(config) {
     });
     MODx.grid.Package.superclass.constructor.call(this,config);
     this.on('render',function() {
-        this.getView().mainBody.update('<div class="x-grid-empty">' + _('loading') + '</div>');
-    },this);
+        this.mask = new Ext.LoadMask(this.body.dom, {
+            msg: _('checking_for_package_updates') + '...'
```

Comment:
Why is the ellipsis separate from the lexicon?

Reply:
The reasoning is so it doesn't need to be included in each language lexicon file, and theoretically the lexicon could be used elsewhere where an ellipses may not be appropriate. 
I see this is not the standard way though so if people would prefer this changed for uniformity sake, I don't mind! :)

Reply:
@muzzwood in some cases it's included in the lexicon, in some it isn't (as far as I've seen). I don't mind the way it is now.

Reply:
It is strange that the formation of the text occurs in the code

`> theoretically the lexicon could be used elsewhere where an ellipses may not be appropriate`
Theoretically do you take into account the peculiarities of RTL-languages?

Reply:
> Theoretically do you take into account the peculiarities of languages (for example, RTL)?

Good point, @muzzwood would you be so kind as to include the ... in the lexicon? Thanks!

Reply:
Sorry for the delay. Very good point @mishantrop , I'll get that changed tonight/tomorrow.